tests/cpp_api/audio_tests/audio_tests.cpp

```diff
@@ -315,7 +332,7 @@ int test(int test_case, const char *path, int qa_mode, int downmix, int gpu) {
             std::cout << "\n ROCAL_DATA_PATH env variable has not been set. ";
             exit(0);
         }
-        if (verify_output(buffer, frames, channels, case_name, max_samples, max_channels, buffer_size, rocal_data_path)) {
+        if (test_case != 8 && verify_output(buffer, frames, channels, case_name, max_samples, max_channels, buffer_size, rocal_data_path)) {
```

Comment:
why test_case 8 is excluded here

Reply:
Test case 8 (NSR) uses a different function for verifying outputs. So we exclude it here and test it with a separate function